### PR TITLE
Restrict attribute models

### PR DESCRIPTION
**Pull request by katia-qm**
1669420302

Background

restrict annotatable type
restrict updating some attribute fields

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,11 @@
 - fixed merge_bulk_responses for the case when no responses are given, therefore uploading media without media_objects isn't identified as an unsuccessful upload anymore [PR#20](https://github.com/quality-match/hari-client/pull/20)
 - fixed bug updating media/annotatable ids for media objects and attributes when shared [PR#27](https://github.com/quality-match/hari-client/pull/27)
 
+### Breaking Changes
+
+- restricted attribute annotatable type only to media, media object and instance [PR#28](https://github.com/quality-match/hari-client/pull/28)
+- restricted fields as `annotatable_id`, `annotatable_type` and `attribute_group` of the attribute to be updated [PR#28](https://github.com/quality-match/hari-client/pull/28)
```
Comment (mylius) [inline CHANGELOG.md:16]: For me it's really hard to get, what's meant here.

Comment (katia-qm) [reply CHANGELOG.md:16]: How would you rephrase it?

Comment (Dens49) [reply CHANGELOG.md:16]: The "breaking changes" section is concerned with breaking changes compared to a previous release.
But Attributes weren't even existing in the previous release of the hari-client (v1.0.0), so IMO these details don't need to be mentioned

Comment (katia-qm) [reply CHANGELOG.md:16]: Oh yeah you're absolutely right! I forgot about it:)

**Review by mylius — APPROVED** [review]

**Pull request merged**